package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shelljs",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
sorry, I do remember you telling me why you chose a patch bump- why was it again? (I'm inclined to go with minor since there's so much at stake :laughing:)